src/search.h

```diff
@@ -53,7 +58,7 @@ struct Search_Cpu : Search {
 
   DeviceSpan<float> next_token_scores_;  // shape (beam_size*batch_size, vocab_size)
 
-  bool done_{};
+  bool done_{false};
```

Comment:
```suggestion
  bool done_{};
```

Don't fight the zero init, embrace it.